### ISSUE TITLE
angular ng-if on I tag does not work

### ISSUE THREAD

**Issue by cadilhac** [FortAwesome/Font-Awesome#12375]
Hi,

I have an angular ng-if directive on a I tag. When it is converted to svg, I can still see it in the DOM on the svg tag (it has been copied) but it doesn't work anymore (not compiled?).
Thanks.

**Comment by knightcode** [comment]
This also seems to be an issue with ngClass, which I've previously been using to dynamically swap between icons like left and right chevrons, up and down arrows. SVGs are just a no-go then? ...saves clock cycles.

**Comment by cadilhac** [comment]
A simple workaround for me is to put the ng-directives on a wrapper element. But this would be better if it could work as is without thinking about it...

**Comment by robmadole** [comment]
Hey @cadilhac 

We have an Angular component that we'd love for you to try out:

https://github.com/FortAwesome/angular-fontawesome

**Comment by cadilhac** [comment]
Great, I will, sure.

**Comment by poddarkhushbu07** [comment]
Any solution to this problem. I am using fontawesome with angular v9

**Comment by tagliala** [comment]
@poddarkhushbu07 does https://github.com/FortAwesome/angular-fontawesome help?

**Comment by poddarkhushbu07** [comment]
@tagliala yes this will work but for that I need to include _FontAwesomeModule_ in my project and  cannot use icons with _i tag of html_. Is there any way to accomplish this with _i tag_. I found a workaround by using ngIf on parent tag of _i tag_

**Comment by tagliala** [comment]
@poddarkhushbu07 are you using CSS implementation or SVG implementation?

**Comment by poddarkhushbu07** [comment]
> @poddarkhushbu07 are you using CSS implementation or SVG implementation?

@tagliala SVG Implementation

**Comment by tagliala** [comment]
@poddarkhushbu07 have you tried with `autoReplaceSvg` set to false?

If it still doesn't work, docs mention that you should use the angular module or use the css implementation

Ref:
- https://fontawesome.com/how-to-use/javascript-api/setup/configuration#autoreplacesvg
- https://fontawesome.com/how-to-use/javascript-api/setup/configuration
- https://fontawesome.com/how-to-use/on-the-web/using-with/angular

**Comment by jacob-silvia-ec** [comment]
For those still struggling with this:

```html
<span *ngIf="condition">
    <i class="fa-solid fa-eye"></i>
</span>
```

**Comment by sensibleworld** [comment]
Thanks for opening this issue! Since it’s been a while and things have changed quite a bit in Font Awesome since this was originally posted, we’re closing it out to help keep our issue tracker focused on current work.

If this is still important for your project, please feel free to re-open or file a fresh issue with updated details. Thanks so much!
